packages/peregrine/lib/store/reducers/cart.js

```diff
@@ -20,7 +21,10 @@ export const initialState = {
 const reducerMap = {
     [actions.getCart.receive]: (state, { payload, error }) => {
         if (error) {
-            return initialState;
+            return {
+                ...initialState,
+                getCartError: payload
```

Comment:
I was going to comment that in the success case we should clear `getCartError` out (set back to `null`), but I notice that none of the other functions do this.

If an initial `getCart` fails and a subsequent retry succeeds, won't `getCartError` stick around with a stale error in it?
That'll lead to problems if we do want to make decisions based on this property.

Reply:
> I was going to comment that in the success case we should clear getCartError out (set back to null), but I notice that none of the other functions do this.

I can do this here :)

Reply:
Done!